src/main/kotlin/com/netflix/dgs/plugin/navigation/SchemaToDataFetcherMarkerProvider.kt

```diff
@@ -34,14 +37,26 @@ class SchemaToDataFetcherMarkerProvider : RelatedItemLineMarkerProvider() {
         val dgsService = element.project.getService(DgsService::class.java)
 
         if (element is GraphQLFieldDefinition) {
-            val dataFetcher = dgsService.getDgsComponentIndex().dataFetchers.find { it.schemaPsi == element }
-
+            val dataFetcher = dgsService.dgsComponentIndex.dataFetchers.find { it.schemaPsi == element }
             if (dataFetcher != null) {
                 val builder =
-                    NavigationGutterIconBuilder.create(IconLoader.getIcon("/icons/dgs.svg", this::class.java))
-                        .setTargets(dataFetcher.psiAnnotation)
-                        .setTooltipText("Navigate to DGS data fetcher")
-                        .createLineMarkerInfo(element)
+                        NavigationGutterIconBuilder.create(IconLoader.getIcon("/icons/dgs.svg", this::class.java))
```

Comment:
Use the icon constant (that's probably the merge conflict)

Reply:
done